clients/drcachesim/standalone_analyzer_example/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
> i5879: Added a standalone example

Please specify what kind of example in the title to make it more descriptive: include "drmemtrace analyzer" or something.